cmd/create.go

```diff
@@ -131,6 +134,8 @@ func (cc *createCmd) run() error {
 		cc.templateWriter = &writers.LocalFSWriter{}
 	}
 
+	cc.repoReader = &readers.LocalFSReader{}
```

Comment:
nope, i had it there since we initialize the templatewriters after the flags are read, but since the reporeader doesn't use any flags we can just move it there

Reply:
ahh so the initConfig only runs when the tool runs fully interactively, since the user has the choice to provide a createConfig in the flag, and the repoReader is being set on the createCommand `cc` not the createConfig `cc`, which I recognize it confusing naming convention. In order to still be able to use default extraction in dryruns where a createconfig is provided, we will need to set the reporeader in the Run method